frontend/src/modules/schemas/editor/components/content/base/GroupRenderer.ts

```
@@ -0,0 +1,209 @@
+        if (!node.hasName(groupName)) {
+          node.addName(groupName);
+          node.on('xChange', updateSize);
+          node.on('yChange', updateSize);
+          node.on('widthChange', updateSize);
+          node.on('heightChange', updateSize);
+        }
+      }
+    }
+
+    function getChildVNodes(vnode: VNode) {
+      return vnode.component!.subTree.children as VNode[];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)